blocks/editable/index.js

```diff
@@ -361,7 +361,7 @@ export default class Editable extends Component {
 		// Find the parent "relative" positioned container
 		const container = this.props.inlineToolbar ?
 			this.editor.getBody().closest( '.blocks-editable' ) :
-			this.editor.getBody().closest( '.editor-visual-editor__block' );
+			this.editor.getBody().closest( '.editor-block-list__block' );
```

Comment:
Is this a good opportunity to stop using a hard-coded selector for this? What are some other options?

Reply:
We could use `context` and some `HOCs` to specify the component to use for anchoring directly as well (rather than relying on DOM hierarchy and selectors).

Reply:
>Is this a good opportunity to stop using a hard-coded selector for this? What are some other options?
>We could also use `context` and some `HOCs` to specify the component to use for anchoring directly as well (rather than relying on DOM hierarchy and selectors).

I definitely agree with the desire here, and hope to discourage these hard-coded selectors from being introduced in the first place (this pull request serving as evidence of the maintenance pain they cause). But I'd rather address these separate from the intent of this pull request.

Reply:
Although given some of the immediate concerns of moving this as raised in https://github.com/WordPress/gutenberg/pull/3546#discussion_r151928760, maybe it is a prime time 😕